src/components/reporter/Form.js

```diff
@@ -318,8 +319,17 @@ const Form = ({ contest, sponsor, repoUrl }) => {
         });
 
         return (
-          <div className={clsx(styles.Form)}>
-            <h1>{sponsor} contest finding</h1>
+          <div className={!isExpended ? clsx(styles.Form) : clsx(styles.Form, styles.FormMax)}>
+            <div className={clsx(styles.FormHeader)}>
+              <h1>{sponsor} contest finding</h1>
+              <button
+                className="button cta-button"
+                type="button"
+                onClick={() => setIsExpended(!isExpended)}
+              >
+                {isExpended ? 'Reduce' : 'Expend'}
```

Comment:
These are not the right words..

I'm struggling to come up with a decent pair, but 'Expend' should definitely be more like 'Expand' or 'Extend'

Reply:
As discussed, will go with icons instead

Reply:
For the code, go with 'isExpanded'; icons won't save us there.

Reply:
Wow that was fast. Thanks team!